libs/sqf/src/analyze/lints/s11_if_not_else.rs

```diff
@@ -0,0 +1,134 @@
+use std::{ops::Range, sync::Arc};
+
+use hemtt_common::config::LintConfig;
+use hemtt_workspace::{
+    lint::{AnyLintRunner, Lint, LintRunner},
+    reporting::{Code, Codes, Diagnostic, Processed, Severity},
+};
+
+use crate::{analyze::SqfLintData, BinaryCommand, Expression, UnaryCommand};
+
+crate::lint!(LintS10IfNotElse);
+
+impl Lint<SqfLintData> for LintS10IfNotElse {
+    fn ident(&self) -> &str {
+        "if_not_else"
+    }
+
+    fn sort(&self) -> u32 {
+        110
+    }
+
+    fn description(&self) -> &str {
+        "Checks for unneeded not"
+    }
+
+    fn documentation(&self) -> &str {
+        r"### Example
+
+**Incorrect**
+```sqf
+if (!alive player) then { player } else { objNull };
+```
+**Correct**
+```sqf
+if (alive player) then { objNull } else { player };
+```
+`!` can be removed and `else` order swapped
+"
+    }
+
+    fn default_config(&self) -> LintConfig {
+        LintConfig::pedantic() // disabled by default
```

Comment:
```suggestion
        LintConfig::help()
            .with_enabled(false)
```
I think Help makes more sense for a severity since it is disabled by default. I also prefer being disabled by default in the lint itself, more clear than a different function and a comment